src/lib/types/restrictions.ts

```diff
@@ -77,7 +85,8 @@ const fixedOutputMetadataValueType = z.object({
   metadataValueType: metadataValueType,
 })
 
-export const userRestrictionValidation = z.object({
+export const userRestrictionValidation = z.array(z.object({
```

Comment:
As the outer of this is now an `array` I don't think any of the internal restriciton types should be. Otherwise it makes the correct interleaving of operations very confusing